.gitmodules

```diff
@@ -1,18 +1,24 @@
 [submodule "tests/lib/forge-std"]
 	path = tests/lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
+	shallow = true
 [submodule "tests/lib/halmos-cheatcodes"]
 	path = tests/lib/halmos-cheatcodes
 	url = https://github.com/a16z/halmos-cheatcodes
+	shallow = true
 [submodule "tests/lib/openzeppelin-contracts"]
 	path = tests/lib/openzeppelin-contracts
 	url = https://github.com/OpenZeppelin/openzeppelin-contracts
+	shallow = true
 [submodule "tests/lib/solmate"]
 	path = tests/lib/solmate
 	url = https://github.com/transmissions11/solmate
+	shallow = true
 [submodule "tests/lib/solady"]
 	path = tests/lib/solady
 	url = https://github.com/Vectorized/solady
+	shallow = true
 [submodule "tests/lib/multicaller"]
 	path = tests/lib/multicaller
 	url = https://github.com/Vectorized/multicaller
+	shallow = true
```

Comment:
what's the effect of this in the normal dev setting for cloning and updating submodules? is this better than having the ci script to use --depth 1 when cloning?

Reply:
not clear, didn't seem to move metrics. It's also weirdly hard to test, because I can't check out this commit directly. I'm not convinced this does anything 😫